config/ant-phase-verify.xml

```diff
@@ -76,6 +76,12 @@
                 file="config/checkstyle-non-main-files-suppressions.xml"/>
       <path>
         <fileset dir="${basedir}" includes="**/*">
+          <!-- Exclude all PNG files globally -->
+          <exclude name="**/*.png"/>
+
+          <!-- Include PNG files from the docs directory -->
+          <include name="docs/**/*.png"/>
```

Comment:
I don't think this include is required, checkstyle should not operate on binary files